src/platform/unix/mod.rs

```diff
@@ -873,50 +922,37 @@ fn recv(fd: c_int, blocking_mode: BlockingMode)
     Ok((main_data_buffer, channels, shared_memory_regions))
 }
 
-#[cfg(target_os="android")]
-fn maybe_unlink(_: *const c_char) -> c_int {
-    // Calling `unlink` on a file stored on an sdcard immediately deletes it.
-    // FIXME: use a better temporary directory than the sdcard via the Java APIs
-    // and threading that value into Servo.
-    // https://code.google.com/p/android/issues/detail?id=19017
-    0
+fn create_string() -> CString {
+    let epoc = UNIX_EPOCH.elapsed().unwrap_or(Duration::new(0, 0));
+    let pid = unsafe {
+        libc::getpid()
+    };
```

Comment:
For such a simple unsafe expression, I believe it's more customary to put it on a single line.